docs/user-guide/cli-install-configure-zosmf.md

```diff
@@ -0,0 +1,33 @@
+# Configuring z/OSMF
+
+Complete the following IBM z/OSMF configuration tasks for your Zowe CLI implementation.
+
+:::info**Required role:** systems programmer
+:::
+
+## Obtaining z/OSMF installation and configuration materials
+
+Before you start the configuration process, review [Overview of z/OSMF](https://www.ibm.com/docs/en/zos/2.2.0?topic=introduction-overview-zosmf) in the IBM Documentation. Use the [First-Time Installation Checklist](https://www.ibm.com/docs/en/zos/2.2.0?topic=introduction-project-plans-configuring-zosmf) from IBM to plan your installation and complete all the required steps.
```

Comment:
@adam-wolfe @zFernand0 -- I couldn't five the v2.5 equivalent for these two articles in the IBM Docs. Could one of you help me find replacements?

Reply:
First link: https://www.ibm.com/docs/en/zos/2.5.0?topic=zosmf-overview
Second link does not appear to have a direct replacement.

Reply:
Yeah, they don't have a consolidated list for a first-time installation

They seem to have broken it down into each subject 😋 
Security has it's own little paragraph about first-time setup, and most services also have it

Here is a quick search for that checklist 😋 
https://www.ibm.com/docs/en/search/First-time%20installation?scope=SSLTBW_2.5.0&tag=izua300